packages/react/src/components/molecules/HeaderNav/main-nav.js

```diff
@@ -223,18 +223,17 @@ export const HeaderNavItem = React.memo(({
   useHeaderNavButtonEffects(buttonRef.current, onButtonLinkClick);
 
   return(
-    <li ref={itemRef} role="none" className={classes} tabIndex="-1">
+    <li ref={itemRef} role="menuitem" className={classes} tabIndex="-1">
       {isCovid ? (
         <a
-          role="menuitem"
           href={href}
           className="ma__main-nav__top-link cv-alternate-style"
           tabIndex="0"
         >
           {text}
         </a>
       ) : (
-        <button ref={buttonRef} type="button" role="menuitem" id={`button${index}`} className="ma__main-nav__top-link" aria-haspopup="true" tabIndex="0" aria-expanded={buttonExpanded}>
+        <button ref={buttonRef} type="button" id={`button${index}`} className="ma__main-nav__top-link" aria-haspopup="true" tabIndex="0" aria-expanded={buttonExpanded}>
```

Comment:
@ygannett but if we keep the menuitem role inside of `<li>` but not overriding the role of button, then `<button>` at the same level as `<a>` will not receive a menuitem role. I feel like that's not a consistent way of applying the roles. To that point, I think it makes sense to apply the menuitem role at the `<li>` level. Thoughts?